spring-boot/proxy/src/main/java/eu/coatrack/proxy/security/ApiKeyFetchingException.java

```diff
@@ -4,7 +4,7 @@
  * #%L
  * coatrack-proxy
  * %%
- * Copyright (C) 2013 - 2020 Corizon | Institut für angewandte Systemtechnik Bremen GmbH (ATB)
+ * Copyright (C) 2013 - 2021 Corizon | Institut für angewandte Systemtechnik Bremen GmbH (ATB)
```

Comment:
Regarding the license, I don't think this will work if we change the value directly.
So the license is generated when you run the mvn command, which means that when a mvn command is executed then the values will go back to `2020`. Please Christoph run it with mvn and check if the year number changed

Reply:
Actually, I haven't changed anything manually within the license. It seems to have changed to 2021 during one of my Maven runs.

Reply:
Ok, then I will resolve conversation, as the update was done automatically on maven